Gemfile

```diff
@@ -25,13 +25,9 @@ group :development, :test do
   gem 'serverspec',              :require => false
   gem 'puppet-lint',             :require => false
   gem 'pry',                     :require => false
+  gem 'pry-byebug',              :require => false
   gem 'simplecov',               :require => false
-  if beaker_version
-    gem 'beaker', *location_for(beaker_version)
-  else
-    gem 'beaker',                :require => false, :platforms => :ruby
-  end
-  gem 'beaker-rspec',            :require => false, :platforms => :ruby
+  gem 'beaker-rspec', *location_for(beaker_rspec_version)
```

Comment:
Does this mean you don't even use beaker or that beaker-rspec takes a dependency on beaker as well?